src/utils/logging.cc

```diff
@@ -65,17 +65,36 @@ void Logging::SetFilename(const std::string& filename) {
 }
 
 LogMessage::LogMessage(const char* file, int line) {
-  using namespace std::chrono;
-  auto time_now = system_clock::now();
-  auto ms =
-      duration_cast<milliseconds>(time_now.time_since_epoch()).count() % 1000;
-  auto timer = system_clock::to_time_t(time_now);
-  *this << std::put_time(std::localtime(&timer), "%m%d %T") << '.'
-        << std::setfill('0') << std::setw(3) << ms << ' ' << std::setfill(' ')
-        << std::this_thread::get_id() << std::setfill('0') << ' ' << file << ':'
-        << line << "] ";
+  *this << FormatTime(std::chrono::system_clock::now()) << ' '
+        << std::setfill(' ') << std::this_thread::get_id() << std::setfill('0')
+        << ' ' << file << ':' << line << "] ";
 }
 
 LogMessage::~LogMessage() { Logging::Get().WriteLineRaw(str()); }
 
+StderrLogMessage::StderrLogMessage(const char* file, int line)
+    : log_(file, line) {}
+
+StderrLogMessage::~StderrLogMessage() {
+  std::cerr << str() << std::endl;
```

Comment:
now that #443 is in, maybe this can be conditional?

Reply:
It's not very trivial (I think it's better to output without timestamp in that case), so I'll do that later.